src/lib/mongoBackend/dbFieldEncoding.cpp

```diff
@@ -41,8 +41,9 @@
 * Replace:
 *   . => =
 */
-std::string dbDotEncode(std::string s)
+std::string dbDotEncode(const std::string& _s)
 {
+  std::string s = _s; // replace cannot be used in const std::string&
   std::replace(s.begin(), s.end(), ESCAPE_1_DECODED, ESCAPE_1_ENCODED);
   return s;
 }
```

Comment:
It depends, of course. 
Do we want to do the replacement in the very string or do we want to do it in a copy?